host_core/lib/host_core/actors/actor_module.ex

```diff
@@ -198,6 +207,7 @@ defmodule HostCore.Actors.ActorModule do
     Logger.info("Subscribing to #{topic}")
     {:ok, subscription} = Gnat.sub(:lattice_nats, self(), topic, queue_group: topic)
     Agent.update(agent, fn state -> %State{state | subscription: subscription} end)
+    Agent.update(agent, fn state -> %State{state | ociref: oci} end)
```

Comment:
You can combine these 2 `Agent.update` calls into one. It's a tiny performance benefit but I think it'll look cleaner.